src/main/scala/PcapExample.scala

```diff
@@ -69,14 +63,16 @@ object PcapExample {
         !(data + offsetBytes + n)
       }
       .foreach { v =>
-        native.stdio.printf(toCString("%02X"), v)
+        native.stdio.printf(c"%02X", v)
       }
     println("...]")
   }
 
-  def main(args: Array[String]): Unit = {
+  def main(args: Array[String]): Unit = Zone { implicit zone => run(args) }
+
+  def run(args: Array[String])(implicit zone: Zone): Unit = {
     val cooked = args.contains("cooked")
-    val live = args.contains("live")
+    val live = !cooked
```

Comment:
Ah .. I thought it was a binary choice, since when I didn't specify live nor cooked, the program just didn't seem to do anything. I'll change it back then.  There wasn't any feedback when the program was run either.   The issue I was having originally was that just running the program with no arguments simply causes an obtuse empty iterator exception.